scripts/firstrun

```diff
@@ -131,7 +131,7 @@ adjust_sunxi_settings() {
 
 do_expand_rootfs() {
 	# get device node for boot media
-	DEVICE="/dev/"$(lsblk -idn -o NAME | grep -w mmcblk0)
```

Comment:
assumption: only one mcc device is available at first boot time, so takes either mmc0 or mmc1
(previously the sdcard on XU4 next was not resized because SDcard is mmc1)